### PR TITLE
Added compliance policies entity

### PR DESCRIPTION
**Pull request by schlupov**

### CODE REVIEW

**Comment by schlupov** [comment]
This isn't the final code. I just needed this because of one cli test.

File: nailgun/entities.py [hunk 1]
```diff
@@ -4966,6 +4969,28 @@ def __init__(self, server_config=None, **kwargs):
         super(PuppetModule, self).__init__(server_config, **kwargs)
 
 
+class Policies(Entity, EntityReadMixin,):
```
Comment (sean797) [inline nailgun/entities.py]: Mind changing the class to `CompliancePolicies`? There could be many types of policies with more plugins.

File: nailgun/entities.py [hunk 2]
```diff
@@ -4966,6 +4969,28 @@ def __init__(self, server_config=None, **kwargs):
...
+        self._fields = {
+            'location': entity_fields.OneToManyField(Location),
+            'name': entity_fields.StringField(
+                required=True,
+                str_type='alpha',
+                length=(4, 30),
+                unique=True
+            ),
+            'organization': entity_fields.OneToManyField(Organization),
+            'hosts': entity_fields.ListField()
```
Comment (jyejare) [inline nailgun/entities.py]:
```suggestion
            'hosts': entity_fields.OneToManyField(Host)
```

File: nailgun/entities.py [hunk 3]
```diff
@@ -4966,6 +4969,28 @@ def __init__(self, server_config=None, **kwargs):
...
+        }
```
Comment (jyejare) [inline nailgun/entities.py]: The compliance Policies has other fields as well, If you add them now it would be helpful. Just a suggestion.

**Comment by schlupov** [comment]
@jyejare @sean797 Changed what was requested.

File: nailgun/entities.py (hunk 3 above)
Comment (schlupov) [reply nailgun/entities.py]: I'll keep it in my mind, I just work on something else now, I'm sorry.

**Pull request merged**